docs/features/link-reference-definitions.md

```diff
@@ -4,7 +4,7 @@ When you use `[[wikilinks]]`, the [foam-vscode](https://github.com/foambubble/fo
 
 This is done to make the content of the file compatible with various Markdown tools (e.g. parsers, static site generators, VS code plugins etc), which don't support `[[wikilinks]]` directly.
 
-This feature is disabled by default because you don't need it to navigate your workspace, it is useful for publishing and compatibility of your notes.
+This feature is enabled by default as it is useful for publishing and compatibility of your notes, but can be disabled because you don't need it to navigate your workspace.
```

Comment:
mmm.. the feature is disabled by default (https://github.com/foambubble/foam-template/blob/7e197a98bd1bd4718b9afc2917c3379c6dca19ac/.vscode/settings.json#L16)

Reply:
Looks like it's disabled in the template, but the default seems to be "withoutExtensions" (like it's written in [Configuration](https://github.com/foambubble/foam-template/blob/master/docs/features/link-reference-definitions.md#configuration) section of the same document)

![image](https://user-images.githubusercontent.com/1502748/134898634-f3bb46a3-ff8e-4773-9df1-819543002880.png)

Reply:
yes, that was done for backwards compatibility, because of the old default
- all new projects using the template would get no definitions
- existing projects that were already using the definitions without special configuration would continue to work the same, but with the option of disabling them

You bring up a good point we might want to disable it by default in the extension as well, the issue for the users who are currently relying on the default value and have the definitions generated.

Reply:
Ok, clear. So I've rephrased it.